apps/api/src/database/seed/data-gs51.js

```diff
@@ -0,0 +1,121 @@
+import { createFolders } from '#repositories/folder.repository.js';
+import { createRepresentation } from './data-test.js';
+
+/**
+ *
+ * @param {import('@prisma/client').PrismaClient} databaseConnector
+ */
+export const createGeneralS51Application = async (databaseConnector) => {
+	try {
+		//1 check if case already exists
+		const caseExists = await databaseConnector.case.findMany({
+			where: { reference: 'GS5110001' }
+		})
+
+		//if we have more than one GS51 case, something went wrong and it will need to be corrected
+		if (caseExists.length > 1) throw new Error('Found more than one GS51 case. There should be only one');
+		
+		//if one case already exists, we can stop here
+		if (caseExists.length == 1) {
+			console.log('GS51 case already exists');
+			return;
+		}
+
+		//2 when case doesn't exists, create sector and subsector for it
+			await databaseConnector.sector.upsert({
+				create: {
+					name: 'general',
+					abbreviation: 'GS51',
+					displayNameEn: 'General',
+					displayNameCy: 'General'
+				},
+				where: { name: 'general' },
+				update: {}
+			});
+
+			await databaseConnector.subSector.upsert({
+				create: { 			
+					name: 'general',
+					abbreviation: 'GS51',
+					displayNameEn: 'Section 51 Advice',
+					displayNameCy: 'Section 51 Advice', 
+					sector: { connect: { name: 'general' } } 
+				},
+				update: {},
+				where: { name: 'general' }
+			});
+
+
+		let	representations = [
+				createRepresentation('GS5110001', 1),
+			];
+			
+		//3 then create the case
+		const GS51_case = await databaseConnector.case.create({
```

Comment:
Yes, since this is a seed script it allows us to create a case with mimial data which works well (I think) as this is just a 'container' case.  In the end, those endpoints call the same prisma queries:
> https://pins-app-back-office-api-dev-ukw-001.azurewebsites.net/api-docs/#/Applications/post_applications

is the create.case 

https://github.com/Planning-Inspectorate/back-office/blob/14fda8b825242082853594b6a8bae9a42468a5c6/apps/api/src/server/repositories/case.repository.js#L162C19-L162C36

and this enpoint
> https://pins-app-back-office-api-dev-ukw-001.azurewebsites.net/api-docs/#/Applications/post_applications__id__start

calls https://github.com/Planning-Inspectorate/back-office/blob/14fda8b825242082853594b6a8bae9a42468a5c6/apps/api/src/server/repositories/folder.repository.js#L228
which I'm calling in the script at https://github.com/Planning-Inspectorate/back-office/blob/46a58abb14d8a5056302533ece813242526f9e39/apps/api/src/database/seed/data-gs51.js#L115